python/pfs/drp/stella/fluxModelInterpolator.py

```diff
@@ -79,38 +155,25 @@ def fromFluxModelData(cls, path: str) -> "FluxModelInterpolator":
         fluxModelInterpolator : `FluxModelInterpolator`
             The model.
         """
-        filePath = os.path.join(path, "interpolator.pickle")
-        if not os.path.exists(filePath):
-            raise RuntimeError(f"'{filePath}' not found. Run `makeFluxModelInterpolator.py` to generate it.")
-        return cls.fromPickle(filePath)
+        interpPath = os.path.join(path, "interpolator.pickle")
+        if not os.path.exists(interpPath):
+            raise RuntimeError(
+                f"'{interpPath}' not found. Run `makeFluxModelInterpolator.py` to generate it."
```

Comment:
Do I have to run `makeFluxModelInterpolator.py` in addition to the `install.py` script? It should be part of the installation process.

Reply:
`makeFluxModelInterpolator.py` was necessary in the past because it took very long time to make it, but now it takes only a few seconds to make the interpolator. I think I can remove `makeFluxModelInterpolator.py`. The installer must not make `interpolator.pickle` because I cannot control the version of python that users use on the installation.